apps/user-portal/src/utils/search-utils.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+/**
+ * Supported operator aliases.
+ *
+ * @type {{ew: string[]; sw: string[]; co: string[]; eq: string[]}}
+ */
+const SUPPORTED_OPERATOR_ALIASES = {
+    co: [ "contain", "contains" ],
+    eq: [ "equal", "equals" ],
+    ew: [ "end with", "ends with" ],
+    sw: [ "start with", "starts with" ]
+};
```

Comment:
Let's have a discussion on these and update on next PR. I think we need use terms like: "startswith:", "endswith:" etc ...

Reply:
Agreed.